src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -11,13 +11,27 @@ import {
 } from 'app/lib/helpers'
 import { paraTimesActions } from 'app/state/paratimes'
 import { AlertBox } from 'app/components/AlertBox'
+import { defaultWithdrawFeeAmount } from 'app/lib/transaction'
+import { StringifiedBigInt } from 'types/StringifiedBigInt'
 import { AmountFormatter } from '../../../components/AmountFormatter'
 import { ParaTimeContent } from '../ParaTimeContent'
 import { ParaTimeFormFooter } from '../ParaTimeFormFooter'
 import { useParaTimes } from '../useParaTimes'
 import { useParaTimesNavigation } from '../useParaTimesNavigation'
 import { FeesSection } from './FeesSection'
 
+function getMaxAmountValue(
+  isDepositing: boolean,
+  accountBalance: StringifiedBigInt,
+  feeAmount: string,
+): StringifiedBigInt {
+  const maxValue =
+    isDepositing && !feeAmount
+      ? accountBalance
+      : BigInt(accountBalance!) - (feeAmount ? BigInt(feeAmount) : BigInt(defaultWithdrawFeeAmount))
```

Comment:
This doesn't seem to match how feeAmount is calculated in buildParaTimeTransfer; and it duplicates its logic